src/saltedhash.h

```diff
@@ -50,4 +50,28 @@ class SaltedOutpointHasher
     }
 };
 
+class SaltedIDHasher
+{
+private:
+    /** Salt */
+    uint64_t k0, k1;
```

Comment:
28dddb91b9744987125cd2248577f38c0b2e77c7
```suggestion
    const uint64_t m_k0, m_k1;
```

Reply:
Done

Reply:
Turns out this can't be const as it causes the `swap` in `LegacyScriptPubKeyMan::EncryptKeys` to not compile.

Reply:
Agree.

> Turns out this can't be const as it causes the `swap` in `LegacyScriptPubKeyMan::EncryptKeys` to not compile.

You mean `LegacyScriptPubKeyMan::Encrypt()` ?